src/app/manager/manager-navigation.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+
+import { CouchService } from '../shared/couchdb.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'planet-manager-navigation',
+  template: `
+    <ul>
+      <li *ngFor="let comp of components"><a [routerLink]="'/' + comp.link">{{comp.name.toUpperCase()}}</a></li>
+      <li><a href="#" class="km-logout" (click)="logoutClick()">LOGOUT</a></li>
+    </ul>
+  `,
+  styleUrls: [ './manager-navigation.scss' ]
+})
+export class ManagerNavigationComponent {
+  constructor(
+    private couchService: CouchService,
+    private router: Router
+  ) {}
+
+  components = [
```

Comment:
This is a copy of the `navigation.component.ts` except for this variable.  Rather than copy the component, you can modify the `navigation.component.ts` to switch between the two navigation sets when the buttons are clicked on the dashboard (using some [custom data binding](https://angular.io/guide/component-interaction)) or by removing the button and bringing that into the navigation bar.